docs/migrating-angular-tutorial.md

```diff
@@ -162,7 +162,7 @@ window.singleSpa.start();
 
 ## Step Four: Adjust your html file
 
-Since most existing SPAs are used to having an index.html file for their css, fonts, third party script-tags, etc., it's likely that you'll have to do some work to make sure all of those keep on working when your SPA becomes an html-less application. 
+Since most existing SPAs are used to having an index.html file for their css, fonts, third party script-tags, etc., it's likely that you'll have to do some work to make sure all of those keep on working when your SPA becomes an html-less application.
```

Comment:
Nit: This sentence took me a bit longer than I anticipated to understand. I'd suggest changing it slightly to something like `Since most existing SPAs depend on having an...`